scheduler/src/cook/mesos/api.clj

```diff
@@ -1369,27 +1370,51 @@
       (when-let [uncoerced-query-params (get-in ctx [:request :query-params "job"])]
         (mapv #(UUID/fromString %) (vectorize uncoerced-query-params)))
       (get-in ctx [:request :body-params :jobs])
-      [(get-in ctx [:request :body-params :job])]))
+      (vectorize (get-in ctx [:request :body-params :job]))))
```

Comment:
Was this broken without the `vectorize`?

Reply:
It wasn't broken before the addition of the `:groups` parameter, but now it's possible for both `:job` and `:jobs` to be empty, in which case we want an empty vector to return here, not `[nil]`.